### PR TITLE
Codegen acos and acosh

### PR DESCRIPTION
**Pull request by wonjoolee95**
Codegen acos and acosh

---

Generated `XLANativeFunctions.cpp`:
```
    at::Tensor XLANativeFunctions::acos(const at::Tensor & self) {
        
        XLA_FN_COUNTER("xla::");
        auto common_device = torch_xla::bridge::GetXlaDevice(self);
        TORCH_INTERNAL_ASSERT(common_device);
        
        torch_xla::XLATensorPtr lazy_self = torch_xla::bridge::GetXlaTensorOrCreateForWrappedNumber(self, *common_device);
        auto out_meta = at::meta::acos(self);
        std::vector<torch::lazy::Shape> shapes{
        torch::lazy::Shape(out_meta.scalar_type(), out_meta.sizes().vec())};
        TORCH_INTERNAL_ASSERT(shapes.size() == 1);
        if(torch::lazy::symbolicShapeEnabled()){
            std::vector<torch::jit::IValue> inputs = { self };
            char* schema_str = "aten::acos(Tensor self) -> Tensor";
            applySymbolicShapesOnLT(schema_str, inputs, shapes);
        }
        
        auto node = torch::lazy::MakeNode<Acos>(lazy_self->GetIrValue(),
                                                                                      std::move(shapes));
        auto result = torch_xla::bridge::AtenFromXlaTensor(
                torch_xla::XLATensor::Create(std::move(node), *common_device));
        return result;
    };

    
    at::Tensor XLANativeFunctions::acosh(const at::Tensor & self) {
        
        XLA_FN_COUNTER("xla::");
        auto common_device = torch_xla::bridge::GetXlaDevice(self);
        TORCH_INTERNAL_ASSERT(common_device);
        
        torch_xla::XLATensorPtr lazy_self = torch_xla::bridge::GetXlaTensorOrCreateForWrappedNumber(self, *common_device);
        auto out_meta = at::meta::acosh(self);
        std::vector<torch::lazy::Shape> shapes{
        torch::lazy::Shape(out_meta.scalar_type(), out_meta.sizes().vec())};
        TORCH_INTERNAL_ASSERT(shapes.size() == 1);
        if(torch::lazy::symbolicShapeEnabled()){
            std::vector<torch::jit::IValue> inputs = { self };
            char* schema_str = "aten::acosh(Tensor self) -> Tensor";
            applySymbolicShapesOnLT(schema_str, inputs, shapes);
        }
        
        auto node = torch::lazy::MakeNode<Acosh>(lazy_self->GetIrValue(),
                                                                                      std::move(shapes));
        auto result = torch_xla::bridge::AtenFromXlaTensor(
                torch_xla::XLATensor::Create(std::move(node), *common_device));
        return result;
    };
```
---
Generated `LazyIr.h`:
```
class Acos : public XlaNode {
 public:
  static torch::lazy::OpKind ClassOpKind() {
    return torch::lazy::OpKind(at::aten::acos);
  }

  Acos(const torch_xla::XlaValue& self, std::vector<torch::lazy::Shape>&& shapes)

      : XlaNode(torch::lazy::OpKind(at::aten::acos),
              {self}, std::move(shapes),
              [&]() { return AcosOutputShape(self); },
              /* num_outputs */ 1,
              torch::lazy::MHash())
        

  {
    
  }

  std::string ToString() const override {
    std::stringstream ss;
    ss << XlaNode::ToString();
    
    return ss.str();
  }

  torch_xla::XlaOpVector Lower(LoweringContext* loctx) const override;

  
  

};

class Acosh : public XlaNode {
 public:
  static torch::lazy::OpKind ClassOpKind() {
    return torch::lazy::OpKind(at::aten::acosh);
  }

  Acosh(const torch_xla::XlaValue& self, std::vector<torch::lazy::Shape>&& shapes)

      : XlaNode(torch::lazy::OpKind(at::aten::acosh),
              {self}, std::move(shapes),
              [&]() { return AcoshOutputShape(self); },
              /* num_outputs */ 1,
              torch::lazy::MHash())
        

  {
    
  }

  std::string ToString() const override {
    std::stringstream ss;
    ss << XlaNode::ToString();
    
    return ss.str();
  }

  torch_xla::XlaOpVector Lower(LoweringContext* loctx) const override;

  
  

};
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,6 +15,9 @@ torch_xla/csrc/XLANativeFunctions.h
 torch_xla/csrc/RegisterXLA.cpp
 torch_xla/csrc/RegisterAutogradXLA.cpp
 
+# Directory autogenerated by full_codegen
+torch_xla/csrc/generated/
```
Comment (wonjoolee95) [inline .gitignore:19]: I noticed the new generated directory getting tracked by git, so I've added to the ignore list. Let me know if this is okay.

Comment (JackCaoG) [reply .gitignore:19]: Thanks!

**Comment by JackCaoG** [comment]
@wonjoolee95 Can you also posted the generated `XLANativeFunction.cpp` and `LazyIr.h` only for the `acos` and `acosh` part. I think this will make review a bit easier.

**Comment by wonjoolee95** [comment]
> @wonjoolee95 Can you also posted the generated `XLANativeFunction.cpp` and `LazyIr.h` only for the `acos` and `acosh` part. I think this will make review a bit easier.

@JackCaoG Good point! Updated the PR summary on top with the two generated files.

**Review by JackCaoG — APPROVED** [review]

**Comment by JackCaoG** [comment]
Thanks Wonjoo!

**Pull request merged**